Makefile

```diff
@@ -359,6 +359,25 @@ test-python-universal-singlestore-online:
  			-k "test_retrieve_online_documents" \
  			sdk/python/tests/integration/online_store/test_universal_online.py
 
+test-python-universal-couchbase-online:
+	PYTHONPATH='.' \
+		FULL_REPO_CONFIGS_MODULE=sdk.python.feast.infra.online_stores.contrib.couchbase_repo_configuration \
+		PYTEST_PLUGINS=sdk.python.tests.integration.feature_repos.universal.online_store.couchbase \
+		python -m pytest -n 8 --integration \
+			-k "not test_universal_cli and \
```

Comment:
Wanted to confirm which ones to skip here?

Reply:
I would just skip what other third party providers do